workshop_app/tests/test_views.py

```diff
@@ -244,3 +260,13 @@ def setUp(self):
 						)
 	
 	
+class TestStaticPages(TestCase):
+
+	def test_testimonials(self): self.client.get('/testimonials/')
```

Comment:
1. Nowhere in PEP8 do they mention this explicitly. I highly suggest you to do your homework before commenting in future.
2. Infact PEP8 has a good section about consistently maintaining style in your code throughout a project/module. Please read this - https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds. Your testing module is inconsistently coded, style wise.
3. This is **most important**, your class <code>TestStaticPages</code> doesn't **test** anything. Please read about Client class in the following link - https://docs.djangoproject.com/en/1.11/topics/testing/tools/